static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -87,6 +92,14 @@ $labour-liverpool-header-element-opacity: .85;
     }
 }
 
+.heroic-immersive{
```

Comment:
@gtrufitt I'm going to untangle these styles and naming this week to be called "Explore" anyway so Labour and Liverpool will be gone. So I'll sort this out then, Simon set all these styles up so it will be a bit of work to untangle. What do you think?